packages/localize/src/tests/runtime.test.ts

```diff
@@ -128,6 +128,24 @@ suite('runtime localization configuration', () => {
       assert.equal(container.textContent, 'Hola friend');
     });
 
+    test('renders multiple parameterized template with different expressions in Spanish', async () => {
```

Comment:
Mind also adding a similar test, but for a parameterized string without the `html` tag? The implementation is very different so this bug shouldn't manifest there, but would be nice to have a test anyway.

Reply:
Thanks! I added more parameterized string template test cases that mirror the html test cases.